pkgs/development/python-modules/rich/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pythonOlder, pytest, pytestcov, typing-extensions, pygments, recommonmark, colorama }:
+
+buildPythonPackage rec {
+  pname = "rich";
+  version = "8.0.0";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "willmcgugan";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0hv27b22x7dbx1i7nzsd8y8fymmvdak2hcx9242jwk4c1a7jr151";
+  };
+
+  postPatch = ''
+   sed -i 's/setuptools.setup(name="rich")/setuptools.setup(name="rich", packages=setuptools.find_packages())/g' setup.py
+  '';
+
+  propagatedBuildInputs = [ typing-extensions pygments recommonmark colorama ];
+
+  checkInputs = [ pytest pytestcov ];
```

Comment:
We generally don't require coverage tests, since they are of no use to us. If you can skip them easily that would be appreciated.

Reply:
seeing as the test suite is invoked through "make test", it may be required there.

However, it would probably be best to look at the command that `make test` is running, and just run it without coverage plugins.

Reply:
Thanks for the feedback. This is the command called: [`pytest --cov-report term-missing --cov=rich tests/ -vv`](https://github.com/willmcgugan/rich/blob/master/Makefile#L2). I guess the `-vv` can be omitted as well? I'll update the PR.